src/petals/server/memory_cache.py

```diff
@@ -172,5 +204,32 @@ def use_cache(self, *handles: Handle) -> Sequence[torch.Tensor]:
         yield tuple(self._allocated_tensors[handle] for handle in handles)
 
 
+@contextlib.asynccontextmanager
+async def wait_for_aenter(context: contextlib.AbstractAsyncContextManager, stop_when_completes: Coroutine):
```

Comment:
This function is not needed since the code using it may be rewritten in a simpler way (see above).

Reply:
it's gone